### PR TITLE
[luci] Verify quantization of Split

### PR DESCRIPTION
**Pull request by jinevening**
This verifies quantization of Split Op.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #6367

### CODE REVIEW

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -267,6 +267,29 @@ template <Type indexT> class SliceTestGraph final : public SimpleTestGraph
   luci::CircleConst *_size = nullptr;
 };
 
+class SplitTestGraph final : public SimpleTestGraph
+{
+public:
+  void init(void)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:273]: please add `override` here :)

Comment (jinevening) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp]: Oops.. Thank you!

**Review by dayo09 — APPROVED** [review]
LGTM :-D

**Comment by jinevening** [comment]
Split has virtual successors (SplitOut), so it is quantized a bit differently from normal operators.

In fp32 model,
fp32 -> Split -> fp32 -> SplitOut -> fp32

In u8 model,
u8 -> Split -> fp32 -> SplitOut -> u8

So the output of SplitOut should be checked.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM

**Pull request merged**